src/pages/about.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { graphql } from 'gatsby';
+import { Page } from '../types';
+import Layout from '../components/Layout';
+import Article from '../components/Article';
+import Footer from '../components/Footer';
+import '../styles/article-reader.scss';
+
+export default function AboutPage({ data }: Page): JSX.Element {
+  const { title, description } = data.page.frontmatter;
+  const { html } = data.page;
+  const { authors } = data.page.fields;
+  return (
+    <>
+      <Layout title={title} description={description} showFooter={false}>
+        <main className="centered-container">
```

Comment:
I think you forgot to add the `centered-container` in styles 😅😅

Reply:
nah I looked at some upcoming styles that will be merged [soon ](https://github.com/nodejs/nodejs.dev/pull/1298/files#diff-0afa2f21875fae40f43e1855da2c0ade6bcd2bfb63bd17efe9b8d924cafc7f1cR529)👍🏻